### PR TITLE
On-release action

### PR DESCRIPTION
**Pull request by antonkurenkov**
Added on-release action for rebuilding and testing images

### CODE REVIEW

**Comment by hanxiao** [comment]
@antonkurenkov could you please sign the CLA?

**Comment by hanxiao** [comment]
why is this workflow bound to master push event? `on_release` means a tag release not every master update.

**Comment by antonkurenkov** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/core-release.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+name: core-release
+
+on:
+  release:
+    types: [published]
```
Comment (hanxiao) [inline .github/workflows/core-release.yml:5]:
```suggestion
on:
  push:
    tags:
      - "v*"  # push to version tags trigger the build
```

**Pull request merged**